README.md

```diff
@@ -11,3 +11,32 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+## Proof that $O(\log_2 n) = O(\log_5 n)$
+
+
+1. **Big-O Definition**: $f(n) \in O(g(n)) \iff \exists c > 0, n_0 \geq 1: f(n) \leq c \cdot g(n) \ \forall n \geq n_0$.
+
+2. **Change of Base Formula**: $\log_a n = \frac{\log_b n}{\log_b a}$.
+
+
+
+## Proof:
+
+To prove $O(\log_a n) = O(\log_b n)$, have to show that $\log_a n \in O(\log_b n)$ and $\log_b n \in O(\log_a n)$. First, considering $\log_a n \in O(\log_b n)$. Using the change of base formula, $\log_a n = \frac{\log_b n}{\log_b a}$. Let $c = \frac{1}{\log_b a} > 0$ (since $\log_b a$ is a positive constant). Then $\log_a n \leq c \cdot \log_b n \ \forall n \geq 1$, which satisfies the definition of $\log_a n \in O(\log_b n)$. 
```

Comment:
The part "...have to show that $\log_a n \in O(\log_b n)$ and $\log_b n \in O(\log_a n)$" is wrong. You're assuming particular T(n) ($log_a n$).

Reply:
Approve this in error. See comment above.